### ISSUE TITLE
SIBR viewer cannot be opened

### ISSUE THREAD

**Issue by dapao-0808** [graphdeco-inria/reduced-3dgs#12]
![12c94bb8b47d34c861d61e6b21e31027](https://github.com/user-attachments/assets/4be53bcc-b914-46e6-9475-4e34e0e5d4fb)
Hi! Thank you very much for your work. I am unable to view my results through SBIR viewer. How can I resolve this issue?

**Comment by YoungXusheng** [comment]
I met this problem as the same, do you solve it?

**Comment by dapao-0808** [comment]
No, this issue may be caused by compatibility with the openGL version. I don't want to change my Nvidia driver, so I plan to install SBIR viewer on another platform and run it with the trained data. However, I currently don't have free time to try it out

**Comment by seigeweapon** [comment]
> No, this issue may be caused by compatibility with the openGL version. I don't want to change my Nvidia driver, so I plan to install SBIR viewer on another platform and run it with the trained data. However, I currently don't have free time to try it out

hi, did you eventially get this resolved? If so, could you please share how you did it? Thanks

**Comment by YoungXusheng** [comment]
您好，我是勇旭升。谢谢您的邮件，我已经收到了。

**Comment by XinyuanNash** [comment]
> 您好，我是勇旭升。谢谢您的邮件，我已经收到了。

Hi,have you resolved the problem?Could you give me some advice?
